site/src/pages/WorkspacesPage/WorkspacesPageView.tsx

```diff
@@ -130,23 +130,21 @@ export const WorkspacesPageView: FC<WorkspacesPageViewProps> = ({ loading, works
 
   return (
     <Margins>
-      <PageHeader
-        actions={
-          <PageHeaderText>
-            Create a new workspace from a{" "}
-            <Link component={RouterLink} to="/templates">
-              Template
-            </Link>
-            .
-          </PageHeaderText>
-        }
-      >
+      <PageHeader>
         <PageHeaderTitle>
           <Stack direction="row" spacing={1} alignItems="center">
             <span>Workspaces</span>
             <WorkspaceHelpTooltip />
           </Stack>
         </PageHeaderTitle>
+
+        <PageHeaderSubtitle>
+          Create a new workspace from a{" "}
```

Comment:
Do these belong in `Language`?

Reply:
I will merge this for now, but I promise I will get back to move this to Language.